### ISSUE TITLE
error when trying to display the number of visit in a partial in backend

### ISSUE THREAD

**Issue by chrisvidal** [dead23angel/octovisit#3]
in my model `fields.yaml` I have:
```
_total_visit:
            label: "Total visit"
            type: partial
            path: ~/plugins/author/xcv/models/lesson/_total_visit.htm
```

in the partial `_total_visit.htm` I have
```
<?= $model->visit(); ?>
```

error thrown:
```
Object of class CWTeam\OctoVisit\PendingVisit could not be converted to string
```

**Comment by dead23angel** [comment]
<?= $model->visit(); ?> returns a collection of visiters, to display the number, use <?= $model->visit()->count(); ?>

**Comment by chrisvidal** [comment]
Please update the documentation then to indicate this is a collection. Maybe rename it to `visitors()` to be more clear?

**Comment by dead23angel** [comment]
Please see the updated documentation, I just released release 1.0.2, the documentation has expanded with examples of displaying the number of views in backend lists and backend forms

**Comment by chrisvidal** [comment]
thanks for the update
